src/config.ts

```diff
@@ -17,6 +17,7 @@ export const config = {
   registrationEndsAt: new Date(process.env.NEXT_PUBLIC_REGISTRATION_END_DATE!),
   reviewEndsAt: new Date(process.env.NEXT_PUBLIC_REVIEW_END_DATE!),
   resultsAt: new Date(process.env.NEXT_PUBLIC_RESULTS_DATE!),
+  votingEndsAt: new Date(process.env.NEXT_PUBLIC_VOTING_END_DATE!),
```

Comment:
This is removed by me from another PR: #124 
Since we fetch the ending date from the contract